src/api/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "api",
+  "version": "1.0.0",
+  "description": "API for project Wayfinder",
+  "main": "./dist/server.js",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node ./dist/server.js",
+    "dev": "concurrently \"npx tsc --watch\" \"nodemon -q dist/index.js\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "dotenv": "^16.0.3",
```

Comment:
I would remove all ^ symbols from your packages.json dependencies. This means it will use version 16.x.x and may break things when updates are made to a package. There is an npm config command that disables this.